dali/pipeline/operators/op_spec.h

```diff
@@ -206,6 +206,11 @@ class DLL_PUBLIC OpSpec {
     return TensorName(outputs_[idx].first, outputs_[idx].second);
   }
 
+  DLL_PUBLIC inline string MutableOutput(int idx) const {
```

Comment:
aren't outputs always mutable?

Reply:
I added this function by mistake, it's a non-working duplicate of the `MutableOutput` below... OutputName is not mutable by default